### ISSUE TITLE
JsonSchemaValidation is missing metadata

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#123]
### Version

0.3.1

### Reproduction steps

1. File `.github/workflows/x.yml` with the following content:
   ```yaml
   on: push
   ```

2. Run `ghlint` with the following arguments:
   ```shell
   ghlint --sarif x.json.sarif .github/workflows/x.yml
   ```

### Expected behavior

Explanation

### Actual behavior

![image](https://github.com/TWiStErRob/net.twisterrob.ghlint/assets/2906988/7fc8ff84-3521-4207-94fa-74b5a9d25b89)
